pipelines/winget-cli-restsource-ci-pr.yml

```diff
@@ -8,51 +8,68 @@ variables:
 - name: DisablePipelineConfigDetector
   value: true
 
-jobs:
-- job: 'BuildPublishTest'
-  displayName: 'Build Publish & Tests'
-  timeoutInMinutes: 60
-  pool:
-    name: 'Azure Pipelines'
-    vmImage: windows-latest
-    demands:
-    - msbuild
-    - visualstudio
-  variables:
-    BuildConfiguration: 'release'
-    BuildPlatform: 'Any CPU'
-
-  steps:
-  # Restore and Build
-  - template: templates/restore-build-publish-test.yml
-
-# Static Analysis Job
-- job: 'StaticAnalysis'
-  displayName: 'Static Analysis'
-  timeoutInMinutes: 60
-  pool:
-    name: 'Azure Pipelines'
-    vmImage: windows-latest
-    demands:
-    - msbuild
-    - visualstudio
-
-  steps:
-  # Analysis
-  - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
-    displayName: 'Run CredScan'
-    inputs:
-      toolMajorVersion: V2
-      debugMode: false
-
-  # Publish SDT Logs
-  - template: templates/copy-and-publish.yml
-    parameters:
-      name: StaticAnalysisLogs
-      source: '$(Agent.BuildDirectory)\_sdt\logs'
-
-  # SDT Post Analysis
-  - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@1
-    displayName: 'Post Analysis'
-    inputs:
-      CredScan: true
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
+      image: windows-2022
+      os: windows
+    customBuildTags:
+    # This is added my 1ES migration tool and it is okay to remove in the future.
+    - ES365AIMigrationTooling
+
+    stages:
+    - stage: Build_Publish_Test_And_GuardianChecks
+      jobs:
+      - job: 'BuildPublishTest'
+        displayName: 'Build Publish & Tests'
+        timeoutInMinutes: 60
+        variables:
+        - name: BuildConfiguration
+          value: 'release'
+        - name: BuildPlatform
+          value: 'Any CPU'
```

Comment:
```suggestion
        BuildConfiguration: 'release'
        BuildPlatform: 'Any CPU'
```

Reply:
I will leave this unchanged for the time being.